packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1422,7 +1442,7 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
         opName: string,
         serializedInterval: ISerializedInterval,
         localSeq: number,
-    ) {
+    ): ISerializedInterval | undefined {
```

Comment:
semantics of undefined should be doc'd since it's not obvious. may also want to leave a reminder comment in the `if (!params)` processing of undefined.